title-2-conservation/eqip_ira/eqip_ira_state_distribution.json

```diff
@@ -0,0 +1,16975 @@
+[
+    {
```

Comment:
Another question about the layout of data structure. Anyway we make the year as the first level (i.e. following what we currently do for title II), then we will have something like:

{
   "2023": [
   "state": "TX"...],
   "2024": ["state":"TX"..]
   ..}

Reply:
There are multiple year information that starts from 2023 to 2031 and some of the years are missing in some states. So probably making like this is easier in term of big picture? If you put 2023, 2024 separate, there is also no room to put 2023 total, maybe we can put under 2023 though but it will probably cause another confusion by that.

Reply:
For the years that are missing, we can skip the corresponding states in this year's JSON. For example, "2023" could include zero input if no state has a prediction for this year.

The benefit of putting years as the first level is that Jonathan mentioned he would like to compare year-by-year. Thus, we will need to implement the feature that retrieves a year's data and compare it with another year's data. The request I may send from the front end could be `.../q=ira&year=2023` or `all_ira_data[2023]` (assume all_ira_data is the big json). It would be easier to derive all years for these kinds of requests.

However, I am open to other solutions as well and would like to hear everyone's thoughts.

Reply:
so let's confirm
you want to make like this

{
"2023": [
"state": "TX"...],
"2024": ["state":"TX"..]
..}

Reply:
Yes, it would be great to have the year as the first level, so something like:

```javascript
{
    "2023": [
      {"state": "TX", totalPayment: xxx, "practices": {...}},
      {"state": "AB", totalPayment: xxx, "practices": {...}},
      ...],
    "2024": [
    {
    {"state": "TX", totalPayment: xxx, "practices": {...}},
    {"state": "AB", totalPayment: xxx, "practices": {...}},
    ...],
    ...
    "2031": [] // assume no state for this year
}
```

Thank you @ywkim312 !